derive_builder_core/src/build_method.rs

```diff
@@ -184,4 +184,27 @@ mod tests {
             }
         ));
     }
+    
+    #[test]
+    fn skip() {
+        let mut build_method = default_build_method!();
+        build_method.enabled = false;
+        
+        assert_eq!(quote!(), quote!(#build_method));
```

Comment:
Could you change the order here for consistency, please?
* All current uses of `assert_eq!` in this crate have the form `assert_eq!(computed_value, expected_value)`.

However, we can talk about switching the argument order _in general_, if you have a strong preference here (maybe as a separate refactoring issue/PR then).